### PR TITLE
[TE] rootcause poc - chart metric split view support

### PR DESCRIPTION
**Pull request by apucher**

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import Ember from 'ember';
 import d3 from 'd3';
 import moment from 'moment';
+import { toBaselineUrn, filterPrefix } from '../../../helpers/utils';
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: nit: let's use absolute path here 'thirdeye-frontend/...'

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 2]
```diff
@@ -115,6 +81,139 @@ export default Ember.Component.extend({
...
+      const { entities, selectedUrns, timeseriesMode } =
+        this.getProperties('entities', 'selectedUrns', 'timeseriesMode');
+
+      if (timeseriesMode != TIMESERIES_MODE_SPLIT) {
+        return {};
+      }
+
+      console.log('rootcauseChart: splitLabels: generating labels');
+      return filterPrefix(selectedUrns, 'thirdeye:metric:')
+        .reduce((agg, urn) => { agg[urn] = entities[urn].label.split("::")[1]; return agg; }, {});
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: nit: let's break this up into multipe line. `entities[urn].label.split("::")[1]; return agg;`

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js (hunk 1 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: fixed

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js (hunk 2 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: fixed

**Pull request merged**